scratch/test_db_counts.py

```
@@ -0,0 +1,21 @@
+
+ROOT_ENV = Path(__file__).resolve().parents[1] / ".env"
+load_dotenv(dotenv_path=ROOT_ENV)
+
+db_url = os.getenv("DATABASE_URL")
+print("Connecting to DB...")
+conn = psycopg2.connect(db_url)
+cur = conn.cursor()
+
+tables = ["tenants", "campaigns", "contacts", "campaign_dispatch", "email_events"]
+for table in tables:
+    cur.execute(f"SELECT COUNT(*) FROM {table};")
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)